backend/src/postgres-adapter/postgres-adapter.service.ts

```diff
@@ -146,4 +146,47 @@ export class PostgresAdapterService {
       client.release();
     }
   }
+
+  async getAllDatabaseConstraints(databaseId: string): Promise<any> {
+    const database = await this.databaseService.findOne(databaseId);
+    const client = await this.getClientForDatabase(databaseId);
+    assert(database, "Database not found");
+
+    try {
+      const result = await client.query(
```

Comment:
May be worth commenting this, at least what the expected Val is cause it's pretty impossible to tell just by reading it